### ISSUE TITLE
Installation stuck at Solving Environment

### ISSUE THREAD

**Issue by Brackarnwar** [rsxdalv/tts-generation-webui#429]
The installation stops at Solving environment. I tried to restart the bat but the same happens. No error.

Applying conda config...
  Checking if Torch is installed...
  Major version update detected. Upgrading base environment
  Using saved GPU choice: NVIDIA GPU
>conda install -y -k conda-forge::uv=0.4.17 conda-forge::vswhere  conda-forge::postgresql=16.4 conda-forge::nodejs=22.9.0 conda-forge::ffmpeg=4.4.2[build=lgpl*] pytorch[version=2.3.1,build=py3.10_cuda11.8*] pytorch-cuda=11.8 torchvision torchaudio cuda-toolkit ninja  -c pytorch -c nvidia/label/cuda-11.8.0 -c nvidia
Channels:
 - pytorch
 - nvidia/label/cuda-11.8.0
 - nvidia
 - defaults
 - conda-forge
Platform: win-64
Collecting package metadata (repodata.json): done
Solving environment: |

**Comment by rsxdalv** [comment]
Hi, sorry that it's freezing there, I suspect that it might be because of the pytorch[version=2.3.1,build=py3.10_cuda11.8*]  version that is causing issues to others as well. I have not found a solution yet though.

**Comment by der-ranjit** [comment]
is there a commit id one could revert to to get it working?

**Comment by rsxdalv** [comment]
> is there a commit id one could revert to to get it working?

No, this is a code that has been unchanged and working for a long time. I'd be more interested in finding out why and how it affects users.
For example, when I tried to reproduce it in Google colab, no issue happened.

**Comment by rsxdalv** [comment]
I hope that this issue has been fixed with the fix that I pushed out last week. I have not seen more reports of this issue, and I was able to confirm the bug and report it to conda team as well; meanwhile implementing a workaround.

**Comment by mstrblueskys** [comment]
> I hope that this issue has been fixed with the fix that I pushed out last week. I have not seen more reports of this issue, and I was able to confirm the bug and report it to conda team as well; meanwhile implementing a workaround.

I am getting this issue when installing for CPU processing.

**Comment by Wontfallo** [comment]
march 7, still seeing this error, tried fresh install in windows. Will attempt in WSL now. 

'''
  Using saved GPU choice: NVIDIA GPU
>conda install -y -k conda-forge::uv=0.4.17 conda-forge::vswhere  conda-forge::postgresql=16.4 conda-forge::nodejs=22.9.0 conda-forge::ffmpeg=4.4.2[build=lgpl*] pytorch[version=2.3.1,build=py3.10_cuda11.8.*] pytorch-cuda=11.8 torchvision torchaudio cuda-toolkit ninja  -c pytorch -c nvidia/label/cuda-11.8.0 -c nvidia
Channels:
 - pytorch
 - nvidia/label/cuda-11.8.0
 - nvidia
 - conda-forge
 - defaults
Platform: win-64
Collecting package metadata (repodata.json): done
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - pytorch==2.3.1[build=py3.10_cuda11.8.*]

Current channels:

  - https://conda.anaconda.org/pytorch
  - https://conda.anaconda.org/nvidia/label/cuda-11.8.0
  - https://conda.anaconda.org/nvidia
  - https://conda.anaconda.org/conda-forge
  - defaults

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.


Error during installation: Command failed with exit code 1
Command failed with exit code 1
Failed to init the app, exiting...
Press any key to continue . . .
'''

**Comment by rsxdalv** [comment]
There's a workaround which involves selecting "pip install" in the GPU
choice and then using the installer_scripts/conda_env_... to open the
virtual environment, where you install the actual version of Pytorch you
want, i.e. 2.3.1 CUDA.
I'm in the process of switching away from conda's pytorch and upgrading to
2.6.0, but it has challenges, especially since not all models support it
yet.

**Comment by rsxdalv** [comment]
The project now uses pip to install pytorch 2.6.0, so these issues should
be gone. Let me know if it still happens, but otherwise I'm closing this
issue.
